core/sys/windows/types.odin

```diff
@@ -801,6 +815,57 @@ CREATESTRUCTW:: struct {
 	dwExStyle:      DWORD,
 }
 
+DEVMODEW :: struct {
+    dmDeviceName:   [32]wchar_t,
+    dmSpecVersion:   WORD,
+    dmDriverVersion: WORD,
+    dmSize:          WORD,
+    dmDriverExtra:   WORD,
+    dmFields:        DWORD,
+	using _: struct #raw_union {
+		// Printer only fields.
+		using _: struct {
+			dmOrientation:   c_short,
+			dmPaperSize:     c_short,
+			dmPaperLength:   c_short,
+			dmPaperWidth:    c_short,
+			dmScale:         c_short,
+			dmCopies:        c_short,
+			dmDefaultSource: c_short,
+			dmPrintQuality:  c_short,
+		},
+		// Display only fields.
+		using _: struct {
+			dmPosition:           POINT,
+			dmDisplayOrientation: DWORD,
+			dmDisplayFixedOutput: DWORD,
+		},
+	},
+    dmColor:       c_short,
+    dmDuplex:      c_short,
+    dmYResolution: c_short,
+    dmTTOption:    c_short,
+    dmCollate:     c_short,
+    dmFormName:    [32]wchar_t,
+    dmLogPixels:   WORD,
+    dmBitsPerPel:  DWORD,
+    dmPelsWidth:   DWORD,
+    dmPelsHeight:  DWORD,
+	using _: struct #raw_union {
+		dmDisplayFlags: DWORD,
+		dmNup:          DWORD,
+	},
+    dmDisplayFrequency: DWORD,
+    dmICMMethod:        DWORD,
+    dmICMIntent:        DWORD,
+    dmMediaType:        DWORD,
+    dmDitherType:       DWORD,
+    dmReserved1:        DWORD,
+    dmReserved2:        DWORD,
+    dmPanningWidth:     DWORD,
+    dmPanningHeight:    DWORD,
+}
```

Comment:
Please make sure the indentation is tabs and consistent

Reply:
I think I fixed it?